src/embed/EmbedVideo.ts

```diff
@@ -21,6 +21,6 @@ export function embedVideo(
 	const cfg = resolveConfig(config);
 
 	const video = new VbrickVideoEmbed(videoId, cfg, el);
-	video.initialize();
+	video.initialize().catch(() => {});
```

Comment:
Log the errors

Reply:
already logged in VbrickEmbed - this is to avoid unhandled promise rejection.